backend/curriculum_tracking/tests/test_creating_cards_and_reviews.py

```diff
@@ -0,0 +1,58 @@
+from django.test import TestCase
+from curriculum_tracking.tests.factories import RecruitProjectFactory
+from curriculum_tracking.models import (
+    AgileCard,
+    ContentItem,
+)
+from . import factories
+from core.tests.factories import UserFactory
+from datetime import timedelta
+
+from curriculum_tracking.constants import (
+    NOT_YET_COMPETENT,
+    COMPETENT,
+    EXCELLENT,
+)
+from django.utils import timezone
+
+
+class CreatingCardsAndReviews(TestCase):
```

Comment:
Yeah this test runs but it will be deleted cause it's not an actual 'test' test, it's just for fooling around and checking how stuff works.  I will delete this once I am happy everything works as it should.